src/MoBi.Engine/MoBi.Engine.csproj

```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-     <PackageReference Include="OSPSuite.Core" Version="11.0.65" />
+	  <PackageReference Include="OSPSuite.Core" Version="11.0.65" />
```

Comment:
indent

Reply:
corrected the first, for the second I have no clue - locally it seems fine...